cursorless-talon/src/cursorless.py

```diff
@@ -15,3 +15,9 @@ def private_cursorless_show_settings_in_ide():
 
     def private_cursorless_show_sidebar():
         """Show Cursorless-specific settings in ide"""
+
+    def private_cursorless_analyze_command_history():
```

Comment:
I think I'd call this `private_cursorless_show_statistics`. Also, not sure this needs an action, but I guess there's not real downside 🤷‍♂️

Reply:
The setting is called command history and I wanted the command to reflect that they belong together? Without a spoken command people will not use this I'm afraid

Reply:
Oh you mean we could just execute this rpc command directly? Yeah you're right we should probably just remove this action

Reply:
Unnecessary private action is now removed.

Reply:
> Idk I find "show statistics" more intuitive. cc/ @josharian you have a leaning?af

My leaning is not to have a spoken command yet, while we mess around to find out what is useful, and then hope that once a handful of us have played a bit, it'll be clearer (a) what we want to show and thus (b) what a good spoken form for that is.

Reply:
Sounds good